landing-page/components/ContactForm.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+export const ContactForm = () => {
+  return (
+    <section className="bg-white">
+      <div className="py-8 lg:py-16 px-4 mx-auto max-w-screen-md">
+        <h2 className="mb-4 text-4xl tracking-tight font-extrabold text-center text-gray-900 dark:text-white">
+          Contact Us</h2>
+        <p className="mb-8 lg:mb-16 font-light text-center text-gray-500 dark:text-gray-400 sm:text-xl">Got a technical
+          issue? Want to send feedback about a beta feature? Need details about our Business plan? Let us know.</p>
+        <form action="#" className="space-y-8">
+          <div>
+            <label htmlFor="email" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your
+              email</label>
```

Comment:
don't know if we want to deal with a lot of different screen sizes at the moment; maybe better to not have any kind of media-queries in here at this time